contrib/inventory/azure_rm.py

```diff
@@ -306,7 +306,7 @@ def __init__(self, args):
         else:
             # try to look up "well-known" values via the name attribute on azure_cloud members
             all_clouds = [x[1] for x in inspect.getmembers(azure_cloud) if isinstance(x[1], azure_cloud.Cloud)]
-            matched_clouds = [x for x in all_clouds if x.name == raw_cloud_env]
+            matched_clouds = [x for x in all_clouds if x.name == raw_cloud_env.name]
```

Comment:
Please also replace the `raw_cloud_env` this in the following context

Reply:
@yuwzho I already applied that change in my fork, tested in the customer environment successfully and I'm just waiting for the approval to merge it to the branch. Thank you.